exercises/concept/zero-zilch-nada/.docs/instructions.md

```diff
@@ -1,40 +1,41 @@
 # Instructions
 
-In this exercise, we will write some simple functions that return zero values for various Go types. Some types
-you might not have seen before. They will be introduced in detail in later exercises.
+In this exercise, we will write some functions that return the zero value for
+Go's types. You may not have seen some of the types yet. They will be
+introduced in detail in later exercises.
```

Comment:
I wonder if it would be possible to have this exercise depend on those other exercises. For example, one could teach booleans first in another exercise without that exercise having to depend on the zero concept.